### ISSUE TITLE
Unknown type: select, render type: select (persistence.recursive)

### ISSUE THREAD

**Issue by SSFGizmo** [Gregor-Agnes/newsletter_subscribe#18]
Plug-In: unsubscribe 

/Configuration/FlexForm/flexform_unsubscribe.xml

```
            <persistence.recursive>
                <TCEforms>
                    <label>LLL:EXT:core/Resources/Private/Language/locallang_general.xlf:LGL.recursive</label>
                    <config>
                        <type>select</type>
                        <items type="array">

                            <numIndex index="2" type="array">
```

Maybe selectSingle like in flexform_subscribe.xml

```
<renderType>selectSingle</renderType>
```

Composer installation
EXT:newsletter_subscribe version 3.2.5 (TYPO3 version 10.4.16, PHP 7.3).

**Comment by Gregor-Agnes** [comment]
Fixed in 84fdc6d

**Comment by Gregor-Agnes** [comment]
Thx for reporting! :)
